IdentityCore/src/MSIDError.h

```diff
@@ -43,6 +43,8 @@ extern NSString * _Nonnull MSIDOAuthErrorDomain;
 extern NSString * _Nonnull MSIDKeychainErrorDomain;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->